components/gcp/dataflow/launch_python/component.yaml

```diff
@@ -18,9 +18,14 @@ description: |
 inputs:
   - {name: python_file_path, description: 'The gcs or local path to the python file to run.'}
   - {name: project_id, description: 'The ID of the parent project.' }
```

Comment:
We can probably use the types in this component yaml?

Reply:
Will do. I have a plan to update all component.yaml in one PR. Maybe I can just use this one.

Reply:
I see. Feel free to do that all in one PR.